### PR TITLE
[Done] Tools misc

### PR DESCRIPTION
**Pull request by Aissaoui-Ahmed**
translate file : `content/community/tools-misc.md`
Ref : [here ](https://github.com/reactjs/ar.reactjs.org/issues/117#issuecomment-518014608)

### CODE REVIEW

File: content/community/tools-misc.md [hunk 1]
```diff
@@ -7,13 +7,13 @@ permalink: community/misc.html
 
 * **[react-localize-redux](https://github.com/ryandrewjohnson/react-localize-redux)** - مكتبة لتوطين React/Redux.
 
-* **[react-inspector](https://github.com/xyc/react-inspector):** إستخدام أداة التطوير لتدقيق في للكائنات داخل تطبيق React.
-* **[jreact](https://github.com/KnisterPeter/jreact):** تصيير React جهة الخادم الجزء الداخلي من JVM (Java 7 or Java 8)
+* **[react-inspector](https://github.com/xyc/react-inspector):** إستخدام أداة التطوير لتدقيق في الكائنات داخل تطبيق React.
```
Comment (3imed-jaberi) [inline content/community/tools-misc.md]:
```suggestion
* **[react-inspector](https://github.com/xyc/react-inspector):** إستخدام أداة التطوير كمدقق للكائنات داخل تطبيق React.
```

File: content/community/tools-misc.md [hunk 2]
```diff
@@ -7,13 +7,13 @@ permalink: community/misc.html
...
+* **[jreact](https://github.com/KnisterPeter/jreact):** تصيير React جهة الخادم في الجزء الداخلي من JVM (Java 7 or Java 8)
```
Comment (3imed-jaberi) [inline content/community/tools-misc.md]:
```suggestion
* **[jreact](https://github.com/KnisterPeter/jreact):** تصيير React من جهة الخادم (SSR) من خلال JVM (Java 7 أو Java 8)
```
or 
```suggestion
* **[jreact](https://github.com/KnisterPeter/jreact):** تصيير React من جهة الخادم (SSR) عبر JVM (Java 7 أو Java 8)
```
or
```suggestion
* **[jreact](https://github.com/KnisterPeter/jreact):** تصيير React من جهة الخادم (SSR) بواسطة JVM (Java 7 أو Java 8)
```

File: content/community/tools-misc.md [hunk 3]
```diff
@@ -7,13 +7,13 @@ permalink: community/misc.html
...
 * **[react-jss](https://github.com/cssinjs/jss/tree/master/packages/react-jss):** إدخال وتنصيب تنيسقات jss في مكونات  React.
```
Comment (3imed-jaberi) [inline content/community/tools-misc.md]: espace :D !! 

```suggestion
* **[react-jss](https://github.com/cssinjs/jss/tree/master/packages/react-jss):** إدخال وتنصيب تنيسقات jss في مكونات  React.
```

File: content/community/tools-misc.md [hunk 4]
```diff
@@ -7,13 +7,13 @@ permalink: community/misc.html
...
 * **[django-react](https://github.com/markfinger/django-react):** تصيير جهة الخادم لمكوّن  React لأجل تطبيقات django.
```
Comment (3imed-jaberi) [inline content/community/tools-misc.md]: better exp. .. 

```suggestion
* **[django-react](https://github.com/markfinger/django-react):** تصيير مكوّنات React من جهة الخادم لـتطبيقات django.
```

File: content/community/tools-misc.md [hunk 5]
```diff
@@ -7,13 +7,13 @@ permalink: community/misc.html
...
 * **[react-engine](https://github.com/paypal/react-engine):** محرك تصيير مركب لتَشاكُلِيّة تطبيقات express لأجل تصييرهما بسهولة واجهة react و واجهة react-router
```
Comment (3imed-jaberi) [inline content/community/tools-misc.md]: <div dir="rtl">
لتَشاكُلِيّة  !! 😕

File: content/community/tools-misc.md [hunk 6]
```diff
@@ -7,13 +7,13 @@ permalink: community/misc.html
...
 * **[react-render-visualizer](https://github.com/redsunsoft/react-render-visualizer):** طريقة مرئية لمعرفة ما هو (إعادة) التصيير ولماذا.
-* **[javascript-monads](https://github.com/dschalk/javascript-monads):** هذا المشورع مكتمل، أصبحت React "الأحادية" أكثر من مجرد لعبة فهو يستعمل بطرق غير تقليدية ، لذا إذا كنت تريد استخدام React بالطريقة التي يقومون بها في Facebook ،هذا ليس لك. شرح الشيفرة لثلاث مصنفات لبيانات المجردة ويمكن تنزيله عن طريق
+* **[javascript-monads](https://github.com/dschalk/javascript-monads):** هذا المشورع مكتمل، أصبحت React "الأحادية" أكثر من مجرد لعبة فهو يستعمل بطرق غير تقليدية ، لذا إذا كنت تريد استخدام React بالطريقة التي يقومون بها في Facebook ،هذا ليس لك. شرح الشيفرة لثلاث  "npm install reactive-monads".مصنفات لبيانات المجردة ويمكن تنزيله عن طريق
```
Comment (3imed-jaberi) [inline content/community/tools-misc.md]:
```suggestion
* **[javascript-monads](https://github.com/dschalk/javascript-monads):** هذا المشورع مكتمل، أصبحت React "الأحادية" أكثر من مجرد لعبة فهو يستعمل بطرق غير تقليدية ، لذا إذا كنت تريد استخدام React بالطريقة التي يقومون بها في Facebook ،هذا ليس لك. يمكن تنزيل التعليمات البرمجية المنقولة لمصنفات البيانات المجردة الثلاثة عن طريق إدخال "npm install reactive-monads".
```

File: content/community/tools-misc.md [hunk 7]
```diff
@@ -7,13 +7,13 @@ permalink: community/misc.html
...
 * **[ReactCSS](http://reactcss.com/):** أساليب التنسيق في JS مع الدعم لـReact.
 * **[ReactQuestionnaire](https://github.com/kouryuu/react-questionnaire):** مكوّنات react بسيطة لبناء إستطلاع أو إستبيان
 * **[renderjs.io](https://renderjs.io/):** محرك البحث الأمثل لـReactjs هي خدمة لصنع تطبيقات reactjs قابلة للتمدد و المشاركة.
```
Comment (3imed-jaberi) [inline content/community/tools-misc.md]:
```suggestion
* **[renderjs.io](https://renderjs.io/):** محرك البحث الأمثل ( SEO ) لـ Reactjs هي خدمة لصنع تطبيقات reactjs قابلة للتمدد و المشاركة.
```

File: content/community/tools-misc.md (hunk 5 above)
Comment (Aissaoui-Ahmed) [reply content/community/tools-misc.md]: > لتَشاكُلِيّة !! 😕

monad

**Pull request merged**